src/xunit.performance.api/XunitPerformanceHarnessOptions.cs

```diff
@@ -15,6 +15,7 @@ internal sealed class XunitPerformanceHarnessOptions
         public XunitPerformanceHarnessOptions()
         {
             _outputDirectory = Directory.GetCurrentDirectory();
+            _tmpDirectory = Directory.GetCurrentDirectory();
```

Comment:
Suggestion: you can use `System.IO.Path.GetTempPath()` to get the tmp dir in a platform-agnostic way.